cassandra/reactive/src/test/java/example/springdata/cassandra/people/ReactivePersonRepositoryIntegrationTest.java

```diff
@@ -89,6 +90,14 @@ void shouldQueryDataWithQueryDerivation() {
 		StepVerifier.create(repository.findByLastname("White")).expectNextCount(2).verifyComplete();
 	}
 
+	/**
+	 * Fetch data limiting result size.
+	 */
+	@Test
+	void limitResultSize() {
+		StepVerifier.create(repository.findByLastname("White", Limit.of(1))).expectNextCount(1).verifyComplete();
```

Comment:
Care to rewrite all `StepVerifier.create(…)` syntax occurrences into `as(StepVerifier::create)` to polish up the code?

Reply:
good catch - sure thing 👍